core/models/asset.py

```diff
@@ -542,6 +545,13 @@ def save(self, *args, **kwargs):
             self.previous_assignee = None
         try:
             super().save(*args, **kwargs)
+            if self.previous_assignee is None:
+                threshold_data = self.asset.model_number.asset_make.asset_type
```

Comment:
This variable would be more descriptive if named `asset_type`. Then you can have asset_type.threshold = ....